src/PHPToJavaScriptTransformer.php

```diff
@@ -223,7 +231,13 @@ protected function transformNull($value)
      */
     protected function escape($value)
     {
-        return str_replace(["\\", "'"], ["\\\\", "\'"], $value);
+        $value = str_replace(["\\", "'"], ["\\\\", "\'"], $value);
+
+        if($this->autoEscapeBreaks == true){
+            $value =  preg_replace("/\r|\n/", "", $value);
```

Comment:
Isn't this just plain removing all newlines, and not escaping them?

I ran into the same problem, this is how I fixed it :) #88